atomic_reactor/constants.py

```diff
@@ -60,3 +60,14 @@
 )
 
 DEFAULT_DOWNLOAD_BLOCK_SIZE = 10 * 1024 * 1024 # 10Mb
+
+LABEL_ALIASES = {
+    "name" : ('name', 'Name'),
```

Comment:
(My preference would be to have a class that knows about which label names to use, yes - similarly to how it is done for eg BuildResponse).